deps/build.jl

```diff
@@ -4,30 +4,48 @@ using BinaryProvider
 const verbose = "--verbose" in ARGS
 const prefix = Prefix(get([a for a in ARGS if a != "--verbose"], 1, joinpath(@__DIR__, "usr")))
 
+# install OpenSSL
+include("build_OpenSSL.jl")
```

Comment:
Alternative for dependencies: https://github.com/JuliaWeb/LibCURL.jl/blob/430047d58c0a0214743af08db114ba8aa1d8f9a6/deps/build.jl#L8

Reply:
Downloading a remote builder is a nightmare when you need to debug